<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Choi/Desktop/인공지능_사진.jpg</normaloff>C:/Users/Choi/Desktop/인공지능_사진.jpg</iconset>
  </property>
  <widget class="QGroupBox" name="GroupBox_dataset">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>811</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Set</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>28</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextEdit" name="text_data_path"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QPushButton" name="button_load_data">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>811</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Result and Graph</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_auc" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>141</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QTextEdit" name="text_result"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_matrix" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_model">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>811</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Select and Predict</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="button_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_load_model">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>621</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="text_model_path"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
